Userland/Services/ImageDecoder/ImageDecoderServer.ipc

```diff
@@ -4,5 +4,5 @@
 endpoint ImageDecoderServer
 {
     decode_image(Core::AnonymousBuffer data) => (bool is_animated, u32 loop_count, Vector<Gfx::ShareableBitmap> bitmaps, Vector<u32> durations)
-    decode_image_with_known_path(DeprecatedString path, Core::AnonymousBuffer data) => (bool is_animated, u32 loop_count, Vector<Gfx::ShareableBitmap> bitmaps, Vector<u32> durations)
+    decode_image_with_known_mime_type(DeprecatedString mime_type, Core::AnonymousBuffer data) => (bool is_animated, u32 loop_count, Vector<Gfx::ShareableBitmap> bitmaps, Vector<u32> durations)
```

Comment:
Can we fold this back into a single `decode_image(Core::AnonymousBuffer data, Optional<DeprecatedString> mime_type)` call? They already do the exact same thing except one passes a hardcoded empty optional.